assets/src/components/admin/admin_add_modal.tsx

```diff
@@ -80,7 +80,11 @@ const defaultAppParamsByAppId = {
     evergreen_content: [],
   },
   elevator_v2: {
-    elevator_id: "ELEVATOR_ID",
+    elevator_id: "",
+    alternate_direction_text: "",
+    accessible_path_direction_arrow: "n",
```

Comment:
Should this `accessible_path_direction_arrow` also have an empty default so that it forces a decision to be made?